New section for Router Protocol (#974)
oasisprotocol/docs: docs/dapp/opl/router-protocol/interface.md

---
description: Router Protocol Interfaces
---
 
# Router Interfaces

> Router Protocol provides a library of `evm-gateway-contracs` to facilitate the

Reviewer: ```suggestion
Router Protocol provides the `evm-gateway-contracts` library to facilitate the
```

> development of cross-chain decentralized applications (dApps).

Reviewer: ```suggestion
development of cross-chain dApps.
```

## Installation

### Using Remix

If you're using [Remix], you can import the contracts directly as shown in the
examples below.

[Remix]: https://remix.ethereum.org/

### Using Hardhat

For Hardhat projects, install the package via npm, yarn or pnpm:

```shell npm2yarn

> npm install @@outerprotocol/evm-gateway-contracts

Reviewer: ```suggestion
npm install @routerprotocol/evm-gateway-contracts
```
